buildtest/cli/report.py

```diff
@@ -91,6 +91,7 @@ def __init__(
         oldest=None,
         count=None,
         pager=None,
+        format_detailed=None,
```

Comment:
please document this variable in docstring

Reply:
fixed